top_100.rb

```diff
@@ -0,0 +1,70 @@
+require 'nokogiri'
+require 'open-uri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.